src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -141,6 +144,27 @@ public CaseDetails getCase(String caseRef, String jurisdiction) {
         }
     }
 
+    public List<Long> getExceptionRecordRefsByEnvelopeId(String envelopeId, String service) {
+        ServiceConfigItem serviceConfig = serviceConfigProvider.getConfig(service);
+
+        String jurisdiction = serviceConfig.getJurisdiction();
+        String caseTypeIdsStr = format("%s_ExceptionRecord", serviceConfig.getJurisdiction());
```

Comment:
Finrem jurisdiction is Divorce. Shouldn't we use `getService` here instead?

Reply:
Well spotted. I mistakenly thought that case type ID was built from jurisdiction. Will correct.